### PR TITLE
Fix for mobile Safari's ReferenceError

### PR DESCRIPTION
**Pull request by amirebrahimi**
Mobile safari throws a ReferenceError even when checking for undefined.

### CODE REVIEW

**Comment by mrdoob** [comment]
Shouldn't performa

**Comment by amirebrahimi** [comment]
Unfortunate, no. Safari throws a ReferenceError even when checking for `performance`.

**Comment by mrdoob** [comment]
How about `window.performance` or `self.performance`?

**Comment by amirebrahimi** [comment]
That's' interesting...

window.performance and self.performance report 'undefined', but performance
throws a ReferenceError.

**Comment by mrdoob** [comment]
I guess we should replace it with `self.performance && self.performance.now` then...?

**Comment by amirebrahimi** [comment]
Went ahead and made those changes and tested that it works.

File: src/Stats.js [hunk 1]
```diff
@@ -147,7 +147,7 @@ var Stats = function () {
 				prevTime = time;
 				frames = 0;
 
-				if ( mem !== undefined ) {
+				if ( mem !== undefined && self.performance ) {
```
Comment (mrdoob) [inline src/Stats.js]: I don't think this one is necessary... Everything else looks good though!

Comment (mrdoob) [inline src/Stats.js]: Actually, I can fix it on my end.

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**